src/Liip/RMT/VCS/VCSInterface.php

```diff
@@ -79,6 +86,14 @@ public function getLocalModifications();
     public function saveWorkingCopy($commitMsg = '');
 
     /**
+     * Revert the last commit. If a message is given, only revert
+     * if the commit message matches.
+     *
+     * @param string|null $commitMsg
+     */
+    public function revertLastCommit($commitMsg = null);
```

Comment:
Yeah, if we start to use user input this could be an error.

But I was not confident enough to just delete the last commit without any check. Can we guarantee that a commit will be done each time ?